packages/editor-ui/src/components/WorkflowSelectorParameterInput/WorkflowSelectorParameterInput.vue

```diff
@@ -81,6 +85,14 @@ const {
 	getWorkflowUrl,
 } = useWorkflowResourcesLocator(router);
 
+const currentProjectName = computed(() => {
+	if (!projectStore?.currentProject || projectStore.currentProject?.type === 'personal') {
```

Comment:
What happens here if users are not having projects feature? It would look weird if it says 'personal'

Reply:
Good catch! Will run it by nick!